### PR TITLE
Expose particles rotation speed option

### PR DESCRIPTION
**Pull request by Lizard-13**
Add the option for minimum and maximum rotation speeds in the IDE, and assume the angle values are always "Mutable" because is the only option now.
The bad point is that I can't test it on Cocos, all the exporters seem to use Pixi...

### CODE REVIEW

**Comment by 4ian** [comment]
> The bad point is that I can't test it on Cocos, all the exporters seem to use Pixi...

You can ;) click here:
image

**Comment by Lizard-13** [comment]
Hehe, my bad, missed that one... you'll think I don't want to test it, but it seems there is something broken with cocos, I get this error opening the `index.html` file on Firefox:
```
                        TypeError: h is null[Saber más] cocos2d-js-v3.10.js:1:24786
_initRenderer     file:///CompilationDirectory/cocos2d-js-v3.10.js:1:24786
prepare           file:///CompilationDirectory/cocos2d-js-v3.10.js:1:21348
prepare/<         file:///CompilationDirectory/cocos2d-js-v3.10.js:1:21785
c                 file:///CompilationDirectory/cocos2d-js-v3.10.js:1:13173
d                 file:///CompilationDirectory/cocos2d-js-v3.10.js:1:13257
cc.initEngine     file:///CompilationDirectory/cocos2d-js-v3.10.js:1:19639
prepare           file:///CompilationDirectory/cocos2d-js-v3.10.js:1:21748
run               file:///CompilationDirectory/cocos2d-js-v3.10.js:1:22021
<anónimo>         file:///CompilationDirectory/main.js:47:1

```
The same error if I build a fresh example, is something broken with the renderer initialization :/

**Comment by 4ian** [comment]
Which version of Firefox? I just tested the platformer example in Firefox and Chrome. Maybe you need to have a web server running? (I run one in the directory of the export with `http-server` (npm install http-server -g))

**Comment by Lizard-13** [comment]
Ha!, it works fine (as always) in Firefox Ubuntu (the other was Firefox Windows).
Now it rotates nicely in Cocos too, also fixed the starting size problem on Cocos: the texture is updated at the beginning and I forgot to use the renderTexture trick in the setTexture function to get a square texture with transparent borders, without this fix the particles sprites were always squared.

File: Extensions/ParticleSystem/particleemitterobject-cocos-renderer.js [hunk 1]
```diff
@@ -273,7 +269,14 @@ gdjs.ParticleEmitterObjectCocosRenderer.prototype.isTextureValid = function(text
 gdjs.ParticleEmitterObjectCocosRenderer.prototype.setTexture = function(texture, runtimeScene){
     var texture = runtimeScene.getGame().getImageManager().getTexture(texture);
     if(texture._textureLoaded){
-        this.renderer.setTexture(texture);
+        var sprite = new cc.Sprite(texture);
```
Comment (4ian) [inline Extensions/ParticleSystem/particleemitterobject-cocos-renderer.js]: Can we add a comment here explaining why this is needed? :)

**Comment by 4ian** [comment]
Looks good, good job! Can you add a quick comment explaining why we need to re-render to a rendertexture for cocos2d-JS? :) Should be good to go then!

**Comment by Lizard-13** [comment]
Added the comment, also now it checks if the texture is squared first, so we don't do it unnecessarily :)

**Comment by 4ian** [comment]
Super, it's helpful! 👍
Good job on this!

**Pull request merged**